src/HashBus.Viewer.Multiple/packages.config

```diff
@@ -5,5 +5,5 @@
   <package id="Humanizer" version="1.37.7" targetFramework="net451" />
   <package id="LiteGuard" version="0.10.0" targetFramework="net451" />
   <package id="Newtonsoft.Json" version="7.0.1" targetFramework="net451" />
-  <package id="RestSharp" version="105.2.3" targetFramework="net451" />
+  <package id="RestSharp" version="105.2.3" targetFramework="net451" requireReinstallation="true" />

```

Comment:
According to [the documentation](https://docs.microsoft.com/en-us/nuget/consume-packages/reinstalling-and-updating-packages) this attribute indicates that we need to uninstall and re-install this package after upgrading the target platform of the project from .NET 4.5.1 to 4.6.1. This is because the package contains a different assembly for use with 4.6.1.

This will have to be done for all the projects which use this package.

Reply:
ran the Update-Package –reinstall, looks good now